qemu/tests/cdrom_block_size_check.py

```diff
@@ -156,7 +159,10 @@ def wait_for_tray_open(qdev):
     def change_media(device, target):
         try:
             with change_check:
-                vm.change_media(device, target)
+                if vm.devices.qemu_version in VersionInterval("[7.1.0, )"):
```

Comment:
Just one question: The version scope `"[7.1.0, )"` is based on the downstream or upstream?
if it is based on the downstream, I think we should leave the changeable interface for the upstream's requirements.
And this is kind of hard code here.
like:
```
        required_qemu_version = self.params["required_qemu_version"]
        if qemu_version in VersionInterval(required_qemu_version):
```
Then we can set the `required_qemu_version` in the cfg according to the different requirements of testing.
Refer to https://github.com/autotest/tp-qemu/blob/96b3f1b35b5d4cc947da47768f87f86fd425df2d/qemu/tests/blockdev_inc_backup_modify_backing_bitmaps.py#L20

Reply:
Updated.